packages/grafana-ui/src/components/Logs/LogRowContextProvider.tsx

```diff
@@ -151,7 +151,7 @@ export const LogRowContextProvider: React.FunctionComponent<LogRowContextProvide
   // React Hook that creates an object state value called result to component state and a setter function called setResult
   // The initial value for result is null
   // Used for sorting the response from backend
-  const [result, setResult] = useState<ResultType>(null as any as ResultType);
+  const [result, setResult] = useState<ResultType>(null as unknown as ResultType);
```

Comment:
```suggestion
  const [result, setResult] = useState<ResultType | null>(null);
```

Reply:
ah thanks, this is much nicer 😍 also exposed a problem in the useEffect which i've now fixed as well 👍